src/utils/WithToggleControl.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+/**
+ * Provides boolean isChecked prop to wrapped component.
+ * Prop name can also be customized if need be
+ *
+ * @param {React.element} ToggleComponent - the component to wrap
+ * @returns {React.element}
+ */
+export default function withToggleControl(WrappedComponent, propName = 'isChecked') {
```

Comment:
I think that comment got buried, but maybe it's already taken care of? It looks like `propName` isn't used any more